images/build_images.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+#####
+# SUPPORTED KAFKA AND SCALA VERSIONS
+#####
+KAFKA_VERSIONS=$(cat supported_kafka.version)
+SCALA_VERSIONS=$(cat supported_scala.version)
+
+#####
+# PRODUCT VERSION
+#####
+PRODUCT_VERSION=$(cat release.version)
+
+#####
+# DOCKER AND PROJECT auxiliary variables
+#####
+DOCKER_VERSION_ARG=$1
+PROJECT_NAME=$2
+DOCKER_TAG=$3
+DOCKERFILE_DIR=$4
+
+echo "Building images with DOCKER_VERSION_ARG=$DOCKER_VERSION_ARG, PROJECT_NAME=$PROJECT_NAME, DOCKER_TAG=$DOCKER_TAG, DOCKERFILE_DIR=$DOCKERFILE_DIR"
+
+#####
+# FOR EACH KAFKA VERSION DOWNLOAD BINARIES
+#####
+for KAFKA_VERSION in $KAFKA_VERSIONS
+do
+    for SCALA_VERSION in $SCALA_VERSIONS
+    do
```

Comment:
This does not work ... you cannot loop through Scala versions if your docker tag doesn't include Scala version as well.